src/bin/commands/images.rs

```diff
@@ -26,6 +26,9 @@ pub struct ListImages {
     /// Container engine (such as docker or podman).
     #[clap(long)]
     pub engine: Option<String>,
+    /// output format
+    #[clap(long, default_value = "human", value_parser = clap::builder::PossibleValuesParser::new(["human", "json"]))]
+    pub output_format: String,
```

Comment:
output is a externally tagged on target, should it maybe be internally tagged?

```json
{
  "aarch64-linux-android": [
    {
      "repository": "ghcr.io/cross-rs/aarch64-linux-android",
      "tag": "local",
      "id": "52c5946ea8ae"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-linux-android",
      "tag": "main",
      "id": "695f693bb38d"
    }
  ],
  "aarch64-unknown-linux-gnu": [
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "0.2.2",
      "id": "7ce08a4ab4f3"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "0.2.4",
      "id": "377343ffa1f4"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "<none>",
      "id": "0575ed604a64"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "local",
      "id": "119d741c8acd"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "local-centos",
      "id": "16f7395d709f"
    },
    {
      "repository": "ghcr.io/cross-rs/aarch64-unknown-linux-gnu",
      "tag": "main",
      "id": "4b4a3b05b155"
    },
    {
      "repository": "localhost/cross-rs/cross-custom-crossissue",
      "tag": "aarch64-unknown-linux-gnu-97a4c",
      "id": "8c4c09ff7f80"
    },
    {
      "repository": "localhost/cross-rs/cross-custom-ring-sqlx",
      "tag": "aarch64-unknown-linux-gnu-c4bb0",
      "id": "d4270573e82c"
    },
    {
      "repository": "localhost/cross-rs/cross-custom-ring-sqlx",
      "tag": "aarch64-unknown-linux-gnu-d5f53",
      "id": "bcd20ef4ea80"
    }
  ]
}
```

Reply:
I think the command name should be changed to either of the following:
- `--message-format` (consistency with `cargo build`).
- `-f`, `--format` (consistency with `docker`).